content/2021/2021-01-22_moderators/index.md

```diff
@@ -0,0 +1,18 @@
++++
+title = "Codewars moderation team"
+description = "Codewars staff reinforced by a team of community members!"
+
+[taxonomies]
+tags = ["announcement", "moderation", "community"]
++++
+
+
+Until recently, Codewars maintainers and admins were overwhelmed by the number of tasks related not only to the maintenance of the platform, fixing bugs, and adding new features but also by issues related to Codewars content and users (poor **@Kazk** \*hug\*). Sometimes it was just too much for them.
```

Comment:
Thanks for doing this.
But like the first post, it sounds really awkward if I (maintainer/admin) post this because it's using third person to talk about myself. Especially "poor me" part.

I'll consider prioritizing adding the author field and letting you guys post under a separate name.

Reply:
My idea was to make the Codewars platform talking, that's why "we", as in "we, the Codewars". Maybe it's just not how it should be done?

If you want to change it to first person voice and post as "me, Kazk, the maintainer", then go ahead, and I can sketch other posts in similar way too.


> I'll consider prioritizing adding the author field and letting you guys post under a separate name.

Well, I do not know about others, but I will definitely not publish anything into public without a proper authorization and language review. I can help with writing as much as I am able to, but I do not feel confident enough to do the whole publication process :)

Reply:
Oh, I always thought `we == The Codewars Team`. Never thought that the platform was talking. Personification? I don't think we do that for blogs, but maybe I'm misunderstanding you.

> If you want to change it to first person voice and post as "me, Kazk, the maintainer", then go ahead, and I can sketch other posts in similar way too.

No, I wouldn't want to focus on myself like that.

> Well, I do not know about others, but I will definitely not publish anything into public without a proper authorization and language review. I can help with writing as much as I am able to, but I do not feel confident enough to do the whole publication process :)

Nothing changes to what you're doing. I just thought it'll be easier for you (and others) to write from your perspective and publish as the moderator team. If it's not, then we'll just continue posting without author.